From cc2211cd57fa11216bed7044afa26c36bec38001 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=D0=BD=D0=B0=D0=B1?= <nabijaczleweli@gmail.com>
Date: Sat, 9 May 2020 15:17:42 +0200
Subject: [PATCH] fix ALL_FILES

This fixes vendoring and patches that remove files when building from
within a git repository

Based on an old version of https://github.com/neomutt/neomutt/pull/2326
---
 Makefile.autosetup | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.autosetup b/Makefile.autosetup
index bfa3e4c..14b10aa 100644
--- a/Makefile.autosetup
+++ b/Makefile.autosetup
@@ -56,7 +56,7 @@ UNINSTALL_TARGETS=	@UNINSTALL_TARGETS@
 
 VPATH=		@VPATH@
 
-ALL_FILES!=	(cd $(SRCDIR) && git ls-files 2>/dev/null) || true
+ALL_FILES!=	(cd $(SRCDIR) && git ls-files | xargs realpath -e) 2>/dev/null || true
 
 default:	all
 
-- 
2.20.1

